docs/pages/eas-update/download-updates.mdx

```diff
@@ -0,0 +1,70 @@
+
+You can disable the default behavior by setting the [`checkAutomatically`](/versions/latest/sdk/updates/#updatescheckautomaticallyvalue) option to `NEVER` in the `updates` configuration. This will prevent the app from checking for updates and downloading them automatically.
+
+</Collapsible>
+
+## Checking for updates while the app is running
+
+You can use `Updates.checkForUpdateAsync()` to check for updates while the app is running. This will return a promise that resolves to a [`UpdateCheckResult` object](/versions/latest/sdk/updates/#updatecheckresult), with `isAvailable` set to `true` if an update is available, and information about the update in the [`manifest`](https://docs.expo.dev/versions/latest/sdk/manifests/#expoupdatesmanifest) property.
+
+If an update is available, you can use the `Updates.downloadAsync()` method to download the update. This will return a promise that resolves when the download is complete. Finally, you can use the `Updates.reloadAsync()` method to reload the app with the new version. The `useUpdates()` hook can also be used to monitor the state of the `expo-updates` package from a React component.
+
+<Collapsible summary="What are common patterns for checking for updates while the app is running?">
+
+- You can check for updates at various points in your app's lifecycle, such as [when it is foregrounded](https://reactnative.dev/docs/appstate) or at some interval. When an update is found, you may want to show a dialog to the user to prompt the user to update.
+- You can check for updates at launch and display your own custom loading screen, if it is very important for your use case to ensure that users always get the latest verison at launch.
+
+</Collapsible>
+
+## Checking for updates while the app is backgrounded
+
+You can use [expo-background-task](https://docs.expo.dev/versions/latest/sdk/background-task/) to check for updates while the app is backgrounded. This is useful for apps that are used in a way that the app is in the background for extended periods of time, such as weather apps or other information apps that are not actively used but are running in the background.
+
+To do this, use the same `Updates.checkForUpdateAsync()` and `Updates.downloadAsync()` methods as you would in the foreground, but execute them inside of a background task.
+
+<Collapsible summary="Should I also apply the update with Updates.reloadAsync() when the app is backgrounded?">
+
+This can be a great way to ensure that the user has the latest version of the app when they open it again. However, it is important to note that, unless you persist the state that the app was in at the time it was backgrounded and restore that state, the user will experience a cold boot when they open the app again. One way to mitigate this is to only do a reload in the background if the app has been inactive for a certain period of time, after which a user is unlikely to expect the app to restore its previous state.
```

Comment:
```suggestion
This can be a great way to ensure the user has the latest version of the app when they open it again. However, it is important to note that, unless you persist the state that the app was in at the time it was backgrounded and restore that state, the user will experience a cold boot when they open the app again. One way to mitigate this is to only do a reload in the background if the app has been inactive for a certain period of time, after which a user is unlikely to expect the app to restore its previous state.
```